VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/Attributes/FlagsEnumValuesAreNotPowersOfTwo/FlagsEnumValuesAreNotPowersOfTwoAnalyzer.cs

```diff
@@ -67,7 +79,7 @@ private void AnalyzeSymbol(SyntaxNodeAnalysisContext context)
 
             var enumName = context.SemanticModel.GetDeclaredSymbol(declarationExpression).Name;
             var enumMemberDeclarations =
-                declarationExpression.ChildNodes().OfType<EnumMemberDeclarationSyntax>().ToList();
+                declarationExpression.ChildNodes().SyntaxNodeOfType<EnumMemberDeclarationSyntax>(SyntaxKind.EnumMemberDeclaration);
```

Comment:
If it's going to be called with an argument (`SyntaxKind`) then it shouldn't be called with a type `T`. I think you can just make the method non-generic